ts/live-compositor/src/hooks/store.ts

```diff
@@ -1,9 +1,7 @@
 export type InputStreamInfo = {
   inputId: string;
-  // TODO: add input stream state
-  // state: 'registered' | 'playing' | 'finished';
-
-  // TODO: add input type (maybe other options too)
+  videoState?: 'ready' | 'playing' | 'finished';
```

Comment:
- not sure about naming
- current API does not allow to distinguish if we have audio and video tracks when registering, (we could sent it as part of the register request response)